cmd/export.go

```diff
@@ -31,7 +31,7 @@ func init() {
 	rootCmd.AddCommand(exportCmd)
 }
 
-const defaultArtifactsDir string = ".ghpc"
+const defaultArtifactsDir string = ".ghpc/artifacts"
```

Comment:
Done. There is a bit of a rat's nest around `modulereader` and `modulewriter` that we need to disentangle some day so that all our packages source a common source of strings like this, but this one works.